src/main/java/pageObject/SeznamPage.java

```diff
@@ -19,7 +18,7 @@ public boolean isOpen() {
     return element().isDisplayed(searchBar);
   }
 
-  public SeznamPage openSeznamPage() throws PageIsNotOpenException {
+  public SeznamPage openSeznamPage() {
     LOG.debug("Opening Seznam in browser");
     getDriver().get("https://www.seznam.cz");
     Asserts.assertIsOpen(this);
```

Comment:
Tyhle asserty bych delal jen kdyz tim treba konci test nebo tak. Neni treba to davat vsude. To spis kosmetickej poznatek. Stejne jako bych odstranil vsechny logy, ktere popisuji nejakou akci, ale nejsou na testu. Na Page by nemel byt log, ktery popisuje nejaky step, jen to tam zneprehlednuje. Log, ktery popisuje step, by mel byt na testu nebo stepu. Idealne na abstract stepu a abstract testu udelat metodu logStep, ktera zaloguje neco ve smyslu "STEP:     <popis stepu>, parametry"